public/locales/es-ES/grafana.json

```diff
@@ -1,5 +1,5 @@
 {
-  "_comment": "Este archivo es la fuente de información para las cadenas en inglés. Edítelo para cambiar los plurales y otras frases de la interfaz de usuario.",
+  "_comment": "Este archivo es la fuente original para las cadenas en inglés. Edita la base del código para cambiar los plurales y otras frases de la interfaz de usuario.",
```

Comment:
Why was this translated into Spanish and deleted in German? Some of the text seems to have changed, but is it not weird that it got deleted?